rule/get_return.go

```diff
@@ -15,12 +15,26 @@ type GetReturnRule struct{}
 func (*GetReturnRule) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
 	var failures []lint.Failure
 
-	onFailure := func(failure lint.Failure) {
-		failures = append(failures, failure)
+	for _, decl := range file.AST.Decls {
+		fd, ok := decl.(*ast.FuncDecl)
+		if !ok {
+			continue
+		}
+
+		if !isGetter(fd.Name.Name) {
+			continue
+		}
+
+		if !hasResults(fd.Type.Results) {
```

Comment:
```suggestion
		if hasResults(fd.Type.Results) {
			continue
		}
```

Also it's easier to read it continues if it has results, instead of the negation